README.md

```diff
@@ -1,3 +1,46 @@
+# Math Magicians
+A Single Page App (SPA) for that manages a book library from the Bookstore API
+
```

Comment:
- [ ] Kindly, make sure that you write the correct title and description of your project.

Reply:
This change has been done well